tests/platform_tests/api/test_fan.py

```diff
@@ -147,10 +177,11 @@ def test_get_fans_target_speed(self, duthost, localhost, platform_api_conn):
     def test_get_fans_speed_tolerance(self, duthost, localhost, platform_api_conn):
 
         for i in range(self.num_fans):
-            speed_tol = fan.get_speed_tolerance(platform_api_conn, i)
-            if self.expect(speed_tol is not None, "Unable to retrieve Fan {} speed tolerance".format(i)):
-                if self.expect(isinstance(speed_tol, int), "Fan {} speed tolerance appears incorrect".format(i)):
-                    self.expect(speed_tol > 0 and speed_tol <= 100, "Fan {} speed tolerance {} reading does not make sense".format(i, speed_tol))
+            speed_tolerance = fan.get_speed_tolerance(platform_api_conn, i)
+            if self.expect(speed_tolerance is not None, "Unable to retrieve Fan {} speed tolerance".format(i)):
+                if self.expect(isinstance(speed_tolerance, int), "Fan {} speed tolerance appears incorrect".format(i)):
+                    self.expect(speed_tolerance > 0 and speed_tolerance <= 100, "Fan {} speed tolerance {} reading does not make sense".format(i, speed_tolerance))
+                    self.compare_value_with_platform_facts('tolernace', speed_tolerance, i)
```

Comment:
Typo: `tolernace`. However,we have decided against adding tolerance values in platform.json, at least for the time being. Suggest removing this check. We can add it later if we change the data contract.